lms/djangoapps/discussion_api/serializers.py

```diff
@@ -191,12 +191,12 @@ class ThreadSerializer(_ContentSerializer):
     closed = serializers.BooleanField(read_only=True)
     following = serializers.SerializerMethodField()
     comment_count = serializers.SerializerMethodField(read_only=True)
```

Comment:
@robrap `comment_count` adds 1 for post itself. It had been in discussion once before that why do we need to add 1 for post itself in comment count (We have had a discussion on [this ticket](https://openedx.atlassian.net/browse/MA-1593) regarding incrementing comment_count with 1 for post.
So we have made a `SerializerMethodField` to add 1 to the comment count returned from `cs_comments_service`
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/discussion_api/serializers.py#L255